internal/collector/iis/iis_web_service.go

```diff
@@ -91,6 +95,10 @@ type perfDataCounterValuesWebService struct {
 	WebServiceTotalSearchRequests                 float64 `perfdata:"Total Search Requests"`
 	WebServiceTotalTraceRequests                  float64 `perfdata:"Total Trace Requests"`
 	WebServiceTotalUnlockRequests                 float64 `perfdata:"Total Unlock Requests"`
+	HttpRequestQueuesCurrentQueueSize             float64 `perfdata:"CurrentQueueSize"`
+	HttpRequestQueuesTotalRejectedRequest         float64 `perfdata:"RejectedRequest"`
+	HttpRequestQueuesMaxQueueItemAge              float64 `perfdata:"MaxQueueItemAge"`
+	HttpRequestQueuesArrivalRate                  float64 `perfdata:"ArrivalRate"`
```

Comment:
```suggestion
	HttpRequestQueuesCurrentQueueSize             float64 `perfdata:"CurrentQueueSize"`
	HttpRequestQueuesTotalRejectedRequest         float64 `perfdata:"RejectedRequest"`
	HttpRequestQueuesMaxQueueItemAge              float64 `perfdata:"MaxQueueItemAge"`
	HttpRequestQueuesArrivalRate                  float64 `perfdata:"ArrivalRate"`
```

A quick google search explains that the counter are part of the `HTTP Service Request Queues` object, but this sub collector take care of `Web Service` only (see line 111). The best way would be a copy of the whole file and create a new sub collector.

Reply:
Done

Reply:
@jkroepke - pls guide me on how to test this locally with help of draft prometheus module?

Reply:
Run `go build -o windows_exporter.exe ./cmd/windows_exporter/` locally to procedure a windows exe file.

Reply:
I don't know how to test it, but its building successfully without error and exe is genreated.